src/selectors/index.js

```diff
@@ -0,0 +1,22 @@
+import { createSelector } from 'reselect';
+
+import type { State } from '../types';
+
+const todosSelector = (state: State) => state.todos;
```

Comment:
I see that your State here is `TodosState & VisibilityFilterState`, but for `todosSelector` we just need `TodosState`, right? In general, for flow types, we should declare type more clearly and specifically.